src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -498,8 +498,34 @@
     },
     "unoWasmBootstrapVersion": {
       "type": "parameter",
-      "datatype": "text",
-      "defaultValue": "7.0.19",
+      "datatype": "text"
+    },    
+    "unoWasmBootstrapVersionDefault": {
+      "type": "generated",
+      "generator": "switch",
+      "replaces": "$WasmBootstrapServer$",
+      "parameters": {
+        "evaluator": "C++",
+        "datatype": "string",
+        "cases": [
+          {
+            "condition": "(tfm == 'net6.0' || tfm == 'net7.0')",
+            "value": "7.0.19"
+          },
+          {
+            "condition": "(tfm == 'net8.0')",
+            "value": "8.0.0-dev.114"
+          }
+        ]
+      }
+    },
+    "unoWasmBootstrapVersionEvaluator": {
+      "type": "generated",
+      "generator": "coalesce",
+      "parameters": {
+        "sourceVariableName": "unoWasmBootstrapVersion",
+        "fallbackVariableName": "unoWasmBootstrapVersionDefault"
```

Comment:
This still doesn't protect against invalid bootstrap versions eg specifying stable 7.x bootstrapper when tfm set to net8